boards/posix/native_posix/main.c

```diff
@@ -57,6 +58,10 @@ void posix_exit(int exit_code)
  */
 void posix_init(int argc, char *argv[])
 {
+	/* Set output to line-buffered even when it is a pipe */
```

Comment:
Do you want to do it just like in the bsim board? 
https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/posix/nrf52_bsim/main.c#L65

Reply:
I'll copy&pasted that, so we match on both platforms.